torch/quantization/quantize_fx.py

```diff
@@ -20,23 +23,36 @@ def fuse_fx(graph_module, inplace=False):
     fuser = Fuser()
     return fuser.fuse(graph_module, inplace)
 
-def _prepare_fx(graph_module, qconfig_dict, inplace, is_standalone_module=False):
+def _prepare_fx(model, qconfig_dict, inplace, is_standalone_module=False):
     r""" Internal helper function for prepare_fx
     Args:
-      `graph_modul`e, `qconfig_dict`, `inplace`: see docs for :func:`~torch.quantization.prepare_fx`
+      `model`, `qconfig_dict`, `inplace`: see docs for :func:`~torch.quantization.prepare_fx`
       `is_standalone_module`: a boolean flag indicates whether we are
       quantizing a standalone module or not, a standalone module
       is a submodule of the parent module that is not inlined in the
 forward graph of the parent module,
       the way we quantize standalone module is described in:
       :func:`~torch.quantization._prepare_standalone_module_fx`
     """
-    _check_is_graph_module(graph_module)
-    graph_module = fuse_fx(graph_module, inplace)
+    # symbolically trace the model
+    standalone_modules = qconfig_dict.get('standalone_module_name', [])
+    class CustomTracer(Tracer):
```

Comment:
Can we move the definition of this class out of the function and instantiate it with standalone_modules as an argument?
```python
class CustomTracer(Tracer):
   def __init__(self, standalone_modules):
      super(CustomTracer,self).__init__()
      self.standalone_modules = standalone_modules

   def is_leaf_module(self, m, module_qualified_name):
            return (m.__module__.startswith('torch.nn') and \
            not isinstance(m, torch.nn.Sequential)) or \
            module_qualified_name in self.standalone_modules

Reply:
sure